modules/hcp-update-using-mce-console.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-updating.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-update-using-mce-console_{context}"]
+= Updating a hosted cluster by using the {mce-short} console
+
+You can update your hosted cluster by using the {mce-short} console.
+
+.Procedure
+
+. Select *All clusters*.
+
+. Navigate to *Infrastructure* -> *Clusters* to view managed hosted clusters.
+
+. Click the *Upgrade available* link to update the control plane and node pools.
+
+The available updates are not fetched from the Cluster Version Operator (CVO) of a hosted cluster. The list of the available updates can be different from the available updates from the following fields of the `HostedCluster` custom resource (CR):
+
+* `status.version.availableUpdates`
+* `status.version.conditionalUpdates`
+
+[IMPORTANT]
+====
+Before updating a hosted cluster, you must refer to the available and conditional updates of a hosted cluster. Choosing a wrong release version might break the hosted cluster.
+====
```

Comment:
We should not include plain text outside a list item in a procedure's steps. Can this be moved to another part of the file or attached to step 3?

Reply:
Addressed.